### DISCUSSION TITLE
How to format date under theme layout ?

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [hexojs/hexo#5185] [Question / Answer] [answered]
i need print `page.updated` date formatted. i tried using `{{ page.updated }}` with nunjucks, but printed only _timestamp_.

source
![image](https://user-images.githubusercontent.com/12471057/230060613-145d4016-58aa-4505-84f2-110662fce236.png)
generated
![image](https://user-images.githubusercontent.com/12471057/230060657-5d314f7b-e362-4206-b327-d088c128ba28.png)

**Comment by stevenjoezhang** [comment, marked as answer]
You can try `{{ date(post.updated) }}` or `{{ full_date(post.updated) }}`
